vcd/sample_vcd_test_config.json

```diff
@@ -24,6 +24,11 @@
     "//": "This section contains the organization composition",
     "org": "myorg",
     "vdc": "myvdc",
+    "providerVdc": {
+      "id": "<uuid>",
```

Comment:
Agree for names here vs UUIDs. Just for consistency reasons with other parts of project.